doc/source/contribution/development/build/unix_cmake.rst

```diff
@@ -69,6 +69,10 @@ Here are descriptions of these options:
     It enables mruby support. The feature isn't enabled by default
     but developers should enable the feature.
 
+.. seealso::
+
+  See :doc:`/install/cmake_options` for more details about CMake options.
+
```

Comment:
How about moving this section's contents to `:doc:/install/others` (and improving the contents with the current `:doc:/install/cmake_options` contents) instead of creating a new `:doc:/install/cmake_options`? We will remove https://groonga.org/docs/install/others.html#configure when we drop support for GNU Autotools.

Reply:
`others.rst`の細かな文言の修正は別にわけましょう。
コンフリクトに関しては先に #1383 を仕上げて、その後これをrebaseしましょうか。新しいファイルを作ってまたすぐ消すというのは（すごく履歴を大事にしているわけじゃないですけど）履歴的にはあんまりやりたくないです。

ちなみに、これはどういう作業をするイメージですか？
これまでの経験を考えると私のイメージしている作業と橋田さんがイメージしている内容が違う可能性が高い気がしています。で、それが発覚するのが橋田さんが一通り作業した後になる可能性が気がしています。

* 私のイメージは、すでにある`unix_cmake.rst`と`windows_cmake.rst`の内容を統合して`others.rst`に移動する。
* `others.rst`のconfigureのセクションと同レベルのセクションとしてCMakeのセクションを作ってそこに↑の統合された内容をまとめる。
* `unix_cmake.rst`と`windows_cmake.rst`の"Run `cmake`"の部分の内容はまるっとなくなって`others.rst`のCMakeのセクションへの参照だけになる

Reply:
> 私のイメージは次の通りです。橋田さんのイメージはどんな感じですか？
> 
> * すでにある`unix_cmake.rst`と`windows_cmake.rst`の内容を統合して`others.rst`に移動する。
> * `others.rst`のconfigureのセクションと同レベルのセクションとしてCMakeのセクションを作ってそこに↑の統合された内容をまとめる。
> * `unix_cmake.rst`と`windows_cmake.rst`の"Run `cmake`"の部分の内容はまるっとなくなって`others.rst`のCMakeのセクションへの参照だけになる

前半の２つについては、同様のイメージでした。
最後の一つについては、（今現在のように）"Run `cmake`"の部分も残しておいて、オプションの詳細は`others.rst`のCMakeのセクションを参照してください、というようなリンクにするイメージでいました。

ただ、改めて考えたのですが、`others.rst`に移動すべきかどうか悩み始めています。
`others.rst`は「インストールにまつわるその他」ではなく、「その他の環境向け」という記事だと思うのですが、実質的にはAutotoolsでのビルドの仕方を解説しています。これを分離して、以下のようにするほうが良いのではないかと考えています。

* others.rst
  * `autotools.rst`と`cmake.rst`への参照を追加して、手動ビルドしてください、という旨の記載にする
  * cmakeのみでビルドするようになったら`autotools.rst`へのリンクは消す。
* autotools.rst
  * Autotoolsでのビルド方法について記載 
  * 今のothers.rstのほぼコピー
  * 将来的にcmakeのみでビルドするようになったら消す。
* cmake.rst
  * cmakeでのビルド方法について記載 
  * 今の`cmake_options.rst`の内容を補強して、（ビルドに必要なライブラリなど）`unix_cmake.rst`と`windows_cmake.rst`を統合した内容にして親切。

いかがでしょうか？

Reply:
このようにすると、既存の`ubuntu.rst`などからのconfigureへのリンクも、`others.rst`へのリンクではなく`autotools.rst`へのリンクになるので、意味的にもしっくりくる気がします。

Reply:
ではそうしましょう。
ちなみに、今の`others.rst`の`configure`のところは見出しだけ残って後は`autotools.rst`へのリンクになるんですよね？（`others.rst`にも`configure`の説明を残したりしないですよね？）

Reply:
> ではそうしましょう。

了解しました。

> ちなみに、今の`others.rst`の`configure`のところは見出しだけ残って後は`autotools.rst`へのリンクになるんですよね？（`others.rst`にも`configure`の説明を残したりしないですよね？）

`others.rst`については、見出しすらも残さなくて良いのではないかと考えていました。
`others.rst`の先頭部分に「その他の環境では手動でビルドする必要があります。Autotoolsでビルドするにはこちらを確認してください。CMakeでビルドするにはこちらを参照してください。」とだけあるようなイメージです。

（`.. _source-configure:`や`.. install-configure-with-match-escalation-threshold:`といったラベルは、`:ref:`で参照しているところ共々適切な名前に変更する必要はあると思います。）


今後の作業は以下のイメージでいます。

* cmake.rstの作成
  * cmake_options.rstをもう少し変更する形でこのPRでやる
* others.rstの内容のautotools.rstへの移動＆others.rstの内容の見直し
  * 別のタスクでやる。
  * すぐできそうなので、このPRの後にやる。

Reply:
ではそうしましょう。

「`cmake.rst`の作成」の中には`unix_cmake.rst`と`windows_cmake.rst`の"Run `cmake`"の中身を削除してリンクにするのも含まれているんですよね？

Reply:
はい、そうします。

Reply:
> （今現在のように）"Run `cmake`"の部分も残しておいて

と書いていますが、既存の内容（`-DCMAKE_INSTALL_PREFIX`の説明とか）は`unix_cmake.rst`と`windows_cmake.rst`に残り続けるんですか？

Reply:
いえ、削除します。

> はい、そうします。

すみません、ここで何についてそうしますと言っていたのか曖昧でした。。。

> `unix_cmake.rst`と`windows_cmake.rst`の"Run `cmake`"の中身を削除してリンクにする

に同意して、削除するつもりです、というつもりでした。。。

Reply:
あ、いや、 https://github.com/groonga/groonga/pull/1403#discussion_r964397076 は https://github.com/groonga/groonga/pull/1403#discussion_r964396259 を見る前に書き込んだだけなので https://github.com/groonga/groonga/pull/1403#discussion_r964396259 がわからなかったら https://github.com/groonga/groonga/pull/1403#discussion_r964397076 を書いたわけではないっす。

Reply:
なるほど！